### PR TITLE
Add notification-2024-09-15-console-maintenance.md

### PR DESCRIPTION
**Pull request by qiffang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by qqqdan — APPROVED** [review]

File: tidb-cloud/notification-2024-09-15-console-maintenance.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+## Maintenance window
+
+- Date: 2024-09-15
+- Start time: 8:00 (UTC+0)
+- End time: 8:10 (UTC+0)
+- Duration: Approximately 10 minutes
+
+> **Note:**
+>
+> Currently, users cannot modify the maintenance timing for the TiDB Cloud console, so you need to plan accordingly in advance. Additionally, over the next 3 months, some users may experience an extra 20-minute maintenance window. We will notify these specific users via email.
```
Comment (qiancai) [inline tidb-cloud/notification-2024-09-15-console-maintenance.md:19]:
```suggestion
> - Currently, users cannot modify the maintenance timing for the TiDB Cloud console, so you need to plan accordingly in advance. 
> - During the next 3 months, some users might experience an additional 20-minute maintenance window. Affected users will receive an email notification in advance.
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: tidb-cloud/notification-2024-09-15-console-maintenance.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+---
+title: 2024-09-15 TiDB Cloud Console Maintenance Notification
+summary: Learn about the details of the TiDB Cloud Console maintenance on September 15, 2024, such as the maintenance window, reason, and impact.
+---
+
+# [2024-09-15] TiDB Cloud console Maintenance Notification
```
Comment (qiancai) [inline tidb-cloud/notification-2024-09-15-console-maintenance.md:6]:
```suggestion
# [2024-09-15] TiDB Cloud Console Maintenance Notification
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**